server.js

```diff
@@ -26,10 +69,58 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({"Hello Traveler!":"Here you can check an awards nominations",
+  Routes: [
+    {
+      "/nominees": "show all nominations",
+      "nominees/category/:category": "show nominations from chosen category"
+    }
+
+  ]});
+});
+
+app.get("/nominees", async (req, res) => {
+ const nominees =  await Nomination.find()
+  res.json(nominees);
 });
 
+app.get("/nominees/category/:category", async (req, res) => {
+  try{
+    const category =  await Nomination.find({category: req.params.category}) // finding a requested category, it is possible to add RegEXP as well inside "find" to make it possible to write only one word.
+                                                                             // example: await Nomination.find({category: new RegExp(req.params.category)})
+    if(category) {
+      res.status(200).json({
+        success: true,
+        status_code: 200,
+        data: category
+      })
+    }
+    else {
+      res.status(404).json({
+        success: false,
+        status_code: 404, 
+        data: "There is not data with this category. Try: <br> best actor <br> best tragic comedy "
+      })
+    }
+  }
+     catch (err) {
+      res.status(400).json({
+        success: false,
+        status_code: 400,
+        error: "Invalid category!! Try: `best actor`, `best tragic comedy`"
+      })
```

Comment:
not sure why i can't see the error messages when entering an invalid category, i only see status code 200 and an empty array []. I think if you change "data" to "error" on line 102 instead it will show the text also? :)